gc/base/SweepHeapSectioning.hpp

```diff
@@ -52,6 +53,7 @@ class MM_SweepHeapSectioning : public MM_BaseVirtual {
 
 	virtual uintptr_t estimateTotalChunkCount(MM_EnvironmentBase* env) = 0;
 	virtual uintptr_t calculateActualChunkNumbers() const = 0;
+	virtual bool isReadyToSweep(MM_EnvironmentBase* env, MM_HeapRegionDescriptor* region) { return false; }
```

Comment:
I guess there is a plan to replace this to abstract as soon as OpenJ9 change is promoted as a part of clean up

Reply:
yes, it is just for avoiding back dependency.